pyqt-apps/siriushla/as_ap_configuration/configuration_window.py

```diff
@@ -0,0 +1,296 @@
+"""Window used to set and get PV values based on a given configuration."""
+import time
+import logging
+from math import isclose
+
+import epics
+from qtpy.QtCore import QThread, Signal, Slot
+from qtpy.QtWidgets import (QComboBox, QDialog, QLabel, QListWidget,
+                            QPushButton, QVBoxLayout, QWidget)
+
+from siriushla.as_ps_cycle.pvnames_tree import PVNameTree
+from siriushla.as_ps_cycle.progress_dialog import ProgressDialog
+from siriushla.widgets.windows import SiriusMainWindow
+from siriuspy.envars import vaca_prefix as _VACA_PREFIX
+from siriuspy.namesys import SiriusPVName
+
+
+class EpicsWrapper:
+    """"""
+
+    def __init__(self, pv):
+        self._pv = epics.get_pv(_VACA_PREFIX + pv)
+
+    @property
+    def pvname(self):
+        """PV Name."""
+        return self._pv.pvname
+
+    def put(self, value):
+        """Put if connected."""
+        if not self._pv.connected:
+            return False
+        if self._pv.put(value):
+            time.sleep(5e-3)
```

Comment:
@gu1lhermelp  I am curious. Why do you need this `sleep`

Reply:
It was really not necessary, removed